takeshape/penny
# Update add to cart events

### Test Plan (Steps to test):

1. Go to a product page on the preview site. Add the item to the cart. The cart should open.
1. Close the open cart. Scroll down to related products. Use quick add to add one to the cart. It should open.
